### PR TITLE
Migrate command `bookieinfo` to bkctl

### PR DESCRIPTION
**Pull request by zymap**
### Motivation

Migrate command `bookieinfo` to bkctl.

### Changes

You can execute as `bin/bkctl bookie  info`.

### CODE REVIEW

**Comment by zymap** [comment]
Ok, I will close this pull request and reopen pull request when i'm done your suggestion. Thank you!

File: tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/BookieCommandGroup.java [hunk 1]
```diff
@@ -40,6 +41,7 @@
         .withParent(BKCtl.NAME)
         .withCategory(CATEGORY_INFRA_SERVICE)
         .addCommand(new LastMarkCommand())
+        .addCommand(new InfoCommand())
```
Comment (sijie) [inline tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/BookieCommandGroup.java]: Actually I just realized that `InfoCommand` is displaying all the informations for the whole cluster, not a single bookie. so I would suggest adding this command as part of `bookies` group. (https://github.com/apache/bookkeeper/blob/master/tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/BookiesCommandGroup.java)

So you might consider adding the command to this location: https://github.com/apache/bookkeeper/tree/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookies

**Comment by zymap** [comment]
@sijie Ok

**Comment by zymap** [comment]
@sijie I want to know is there any config for IDE to check style? I import the config `formatter.xml`, but still not work for importorder. Any suggestion?

**Comment by sijie** [comment]
@zymap 

you can try the steps in [bookkeeper contribution guide](http://bookkeeper.apache.org/community/contributing/#checkstyle).

you can run `mvn checkstyle` to verify if there is any checkstyle issue.

**Comment by zymap** [comment]
@sijie thx!

**Comment by zymap** [comment]
run bookkeeper-server replication tests

**Comment by zymap** [comment]
run bookkeeper-server replication tests

**Pull request merged**